node/src/chain_spec/neumann.rs

```diff
@@ -119,10 +120,28 @@ pub fn local_testnet_config() -> ChainSpec {
 			let endowed_accounts: Vec<(AccountId, Balance)> =
 				accounts.iter().cloned().map(|k| (k, initial_balance)).collect();
 
-			let vesting_json =
-				&include_bytes!("../../../distribution/neumann_vesting_local.json")[..];
```

Comment:
Opted to do vesting config in rust so that timestamps can be dynamic and in the future. The timestamps in this file are from march. I wasn't sure who owned the accounts in the json file so opted to use the dev seeds (alice, bob, etc) for vesting too.